docs/OMF.rst

```diff
@@ -165,6 +165,9 @@ The *Basic* tab contains the most commonly modified items
     When using the PI Buffer Subsystem, set the action code to *create* which will allow new data to be compressed normally.
     One disadvantage of the *create* action code is that multiple values with the same timestamp will all be stored. 
 
+  - **Enable OMF Logging**: The Enable OMF Logging flag allows users to toggle the logging functionality on or off.
```

Comment:
This should probale not be described as logging as this will cause confusion with the logging we do to the syste log. We should probably call this detailed tracing of OMF messages. We shoudl also say the fiel created will be collected by the support bundle or may be viewed in the filesystem under the data directory.